data/map planets.txt

```diff
@@ -4947,6 +5406,28 @@ planet "Vivid Aether"
 	description `	The life-bearing layer only extends downward a few hundred kilometers before pressures rise and the temperature becomes inhospitable to life. Deeper down, the atmosphere is barren and lifeless.`
 	government Uninhabited
 
+planet Vyraa-Giiq-Sora
+	attributes boldtourism successor uninhabited
+	landscape land/myrabella0
+	description `It's not clear why this planet remains uninhabited, as it possesses a breathable atmosphere and temperate climate; the surface is covered in indigenous lifeforms, many of which exhibit some degree of bioluminescence.`
+		to display
+			not "known to the successors"
+	description `Despite the verdant and often bioluminescent life on its surface, the atmosphere of this planet is highly toxic to the Successors' biochemistry, and so they have made no permanent settlements here. While it is occasionally visited by the odd few tourists who don't mind having to wear an isolation suit, its distance from the majority of Successor space means that the Successors largely leave it alone.`
+		to display
+			has "known to the successors"
+	government Uninhabited
+
+planet Vyraa-Viir-Kella
+	attributes successor uninhabited
+	landscape land/water4
+	description `This planet has a clear, oxygen-rich atmosphere, and a cool and watery surface with many large lakes. There appear to be no sites of permanent habitation, though a couple of landing pads and associated temporary shelters have been built on islands in the southern hemisphere.`
+		to display
+			not "known to the successors"
+	description `This planet is being surveyed to establish its potential as a new settlement location for the New Houses. Although its gravity is somewhat high for Successor tastes, it has a clear, oxygen-rich atmosphere, and a cool and watery surface. The first few settlers are scheduled to arrive here in a few years, once the planetary surveys are complete and have declared it safe and geologically stable.`
```

Comment:
```suggestion
	description `This planet is being surveyed to establish its potential as a new settlement location for the New Houses. Although its gravity is somewhat high for Successor tastes, it has a clear, oxygen-rich atmosphere, and a cool and watery surface. The first handful of settlers are scheduled to arrive here in a few years, once the planetary surveys are complete and have declared it safe and geologically stable.`
```

Reply:
I've tried to avoid hand-related language in the Successor descriptions because the Successors don't have hands.

Reply:
Good point.
Is there another synonym you can use, so that the duplicate usage of the word "few" is avoided?